<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QMainWindow" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-uncertain">
    <normaloff>../../../Downloads/clueless.ico</normaloff>../../../Downloads/clueless.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPass">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Customer Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoginExit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegister">
    <property name="title">
     <string>Register</string>
    </property>
    <addaction name="actionRegister"/>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
    <addaction name="actionAdmin_Login"/>
   </widget>
   <addaction name="menuRegister"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegister_Account">
   <property name="text">
    <string>Register Account</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Create Account</string>
   </property>
  </action>
  <action name="actionAdmin_Login">
   <property name="text">
    <string>Admin Login</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoginExit</sender>
   <signal>clicked()</signal>
   <receiver>LoginForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLogin</sender>
   <signal>clicked()</signal>
   <receiver>LoginForm</receiver>
   <slot>on_btnLogin_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRegister</sender>
   <signal>triggered()</signal>
   <receiver>LoginForm</receiver>
   <slot>newRegister()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdmin_Login</sender>
   <signal>triggered()</signal>
   <receiver>LoginForm</receiver>
   <slot>adminLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnLogin_clicked()</slot>
  <slot>newRegister()</slot>
  <slot>adminLogin()</slot>
 </slots>
</ui>
